arch/arm64/Kconfig

```diff
@@ -44,6 +44,13 @@ config ARCH_CHIP_QEMU
 	---help---
 		QEMU virt platform (ARMv8a)
 
+config ARCH_CHIP_FVP_ARMV8R
+	bool "ARM FVP virt platform (ARMv8r)"
+	select ARCH_CORTEX_R82
+	select ARCH_HAVE_ADDRENV
+	select ARCH_NEED_ADDRENV_MAPPING
```

Comment:
r82 doesn't have mmu, should we select ARCH_HAVE_ADDRENV and ARCH_NEED_ADDRENV_MAPPING?